src/metabase/api/timeline.clj

```diff
@@ -15,13 +15,16 @@
   "Events Query Parameters Schema"
   (s/enum "events"))
 
+(def Icons
```

Comment:
I remember thinking about this location a bit. This seems like it specs the model not just the api. I think it should live in whichever model does not depend on the other model